### PR TITLE
add verification for KUBE_CA contents

### PR DESCRIPTION
**Pull request by gabemontero**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1548619

@openshift/sig-developer-experience ptal

### CODE REVIEW

File: 2/contrib/jenkins/kube-slave-common.sh [hunk 1]
```diff
@@ -74,6 +74,11 @@ function generate_kubernetes_config() {
     [ -z "$oc_cmd" ] && return
     [ ! has_service_account ] && return
     local crt_contents=$(cat "${KUBE_CA}")
+    # do not dump contents of cert here, but allow any error messages to be seen
+    openssl x509 -in $KUBE_CA > /dev/null
+    if [ $? -eq 1 ] ; then
+      crt_contents=""
```
Comment (bparees) [inline 2/contrib/jenkins/kube-slave-common.sh]: i take it setting an empty cert in the xml is not a problem?  if so, lgtm.

Comment (gabemontero) [reply 2/contrib/jenkins/kube-slave-common.sh]: yep good question @bparees ....,I have verified by setting the cert field to empty and then restarted Jenkins.  It came up fine.  No exceptions.

**Comment by gabemontero** [comment]
oops there was some feedback in the bugzilla that I was about to push .... I now realize @bparees was not copied on the bugzilla 

I'll create a follow up PR .... this change as-is it benign

**Pull request merged**